doc/development.md

```diff
@@ -2,7 +2,7 @@
 
 Requirements:
   - Go 1.10+
```

Comment:
This needs to be changed to Go 1.13+

Reply:
We can use go version above 11 with go modules just that we need to set the GO111MODULE=on in case we are inside the go path to use go modules I have mentioned that in the document as well